src/app/service/poap/ClaimPOAP.ts

```diff
@@ -0,0 +1,58 @@
+import { CommandContext } from 'slash-create';
+import { GuildMember, MessageEmbed } from 'discord.js';
+import { Collection, Cursor, Db } from 'mongodb';
+import MongoDbUtils from '../../utils/MongoDbUtils';
+import constants from '../constants/constants';
+import { POAPUnclaimedParticipants } from '../../types/poap/POAPUnclaimedParticipants';
+import Log from '../../utils/Log';
+import ServiceUtils from '../../utils/ServiceUtils';
+import POAPUtils from '../../utils/POAPUtils';
+
+const ClaimPOAP = async (ctx: CommandContext, guildMember: GuildMember, platform: string, code: string): Promise<any> => {
+	Log.debug(`starting claim with claimCode: ${code}`);
+	
+	POAPUtils.validateClaimCode(code);
+	
+	if (platform == constants.PLATFORM_TYPE_TWITTER) {
+		await ctx.send('Twitter platform is not supported at this time. Please reach out to community organizer for missing POAP');
```

Comment:
'So, the Twitter platform isn't supported with what you\'re trying to do here. Reach out to community organizer to hook you up with your POAP.'

Reply:
Oops missed this